src/routes/app/+page.svelte

```diff
@@ -33,6 +39,22 @@
       $selected = [];
     }
   }
+
+  // filter out deleted documents that haven't been purged from search yet
+  function excludeDeleted(
+    searchResults: DocumentResults,
```

Comment:
This is a really smart move!

Reply:
The current site does a version of this, too. Solr ends up being _just_ enough behind that you'll notice if you delete a document and go right back to search results. It'll probably be gone by the next refresh, so this should be enough.